docs/whats-new.md

```diff
@@ -4,30 +4,55 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
+- A suite of new widgets have been added to the [`@deck.gl/widgets`](./api-reference/widgets/overview.md) module:
+  - [ContextMenuWidget](./api-reference/widgets/context-menu-widget.md)
+  - [FpsWidget](./api-reference/widgets/fps-widget.md)
+  - [GeocoderWidget](./api-reference/widgets/geocoder-widget.md)
+  - [GimbalWidget](./api-reference/widgets/gimbal-widget.md)
+  - [InfoWidget](./api-reference/widgets/info-widget.md)
+  - [LoadingWidget](./api-reference/widgets/loading-widget.md)
+  - [ResetViewWidget](./api-reference/widgets/reset-view-widget.md)
+  - [ScaleWidget](./api-reference/widgets/scale-widget.md)
+  - [ScreenshotWidget](./api-reference/widgets/screenshot-widget.md)
+  - [SplitterWidget](./api-reference/widgets/splitter-widget.md)
+  - [StatsWidget](./api-reference/widgets/stats-widget.md)
+  - [ThemeWidget](./api-reference/widgets/theme-widget.md)
+  - [TimelineWidget](./api-reference/widgets/timeline-widget.md)
+  - [ViewSelectorWidget](./api-reference/widgets/view-selector-widget.md)
+- Pre-wrapped React components - deck.gl widgets are available via the [`@deck.gl/react`](./api-reference/react/overview.md) package
+- Custom widgets in pydeck - via the `custom_libraries` parameter, Python users can integrate custom deck.gl widgets seamlessly.
+- Documentation for styling and custom themes - see [Styling Widgets](./api-reference/widgets/styling).
 
-### Widgets
+### WebGPU Early Preview
 
- A suite of new widgets have been added to the [`@deck.gl/widgets`](./api-reference/widgets/overview.md) module:
+A few deck.gl layers can now be run on WebGPU in the website, by selecting the `WebGPU` tab:
 
-- [ResetViewWidget](./api-reference/widgets/reset-view-widget.md)
-- [ScaleWidget](./api-reference/widgets/scale-widget.md)
-- [GeocoderWidget](./api-reference/widgets/geocoder-widget.md)
-- [ScreenshotWidget](./api-reference/widgets/screenshot-widget.md)
-- [LoadingWidget](./api-reference/widgets/loading-widget.md)
-- [ThemeWidget](./api-reference/widgets/theme-widget.md)
-- [InfoWidget](./api-reference/widgets/info-widget.md)
-- [SplitterWidget](./api-reference/widgets/splitter-widget.md)
+- [LineLayer](../examples/line-layer)
+- [PointCloudLayer](../examples/point-cloud-layer)
+- [ScatterplotLayer](../examples/scatterplot-layer)
 
-Pre-wrapped React components for the new deck.gl widgets are available via the [`@deck.gl/react`](./api-reference/react/overview.md) package.
+See documentation about how to [test WebGPU support](./developer-guide/webgpu.md).
 
 ### Core
 
-- [`View.clone()`](./api-reference/core/view.md) - New method that simplifies creating new Views with modified props, similar to `Layer.clone()`.
+- [`View.clone()`](./api-reference/core/view.md) - New method that simplifies creating new Views with modified props, similar to `Layer.clone()`
+- Async DeckPicker methods - New asynchronous methods for non-blocking operations and (depending on platform and usage) improved picking performance.
```

Comment:
@ibgreen Correct me if I'm wrong, but these aren't in the public API yet. Async picker methods are [implemented](https://github.com/visgl/deck.gl/blob/71d8479ba964b5bb38529e45a469f1f522cb12ee/modules/core/src/lib/deck-picker.ts#L109-L110).. but I don't see then exposed yet on [`Deck`](https://github.com/visgl/deck.gl/blob/71d8479ba964b5bb38529e45a469f1f522cb12ee/modules/core/src/lib/deck.ts#L722-L723).

I think we have two options here.. keep this as an undocumented change for now, or add new `pick...Async()` to `Deck` and document it in near [`pickObject`](https://github.com/visgl/deck.gl/blob/71d8479ba964b5bb38529e45a469f1f522cb12ee/docs/api-reference/core/deck.md#L637)